typescript/web/src/components/datasets/__tests__/upsert-dataset-modal.tsx

```diff
@@ -68,6 +69,11 @@ test("should enable start button when dataset name is not empty", async () => {
 
 test("should create a dataset when the form is submitted", async () => {
   const onClose = jest.fn();
+  // Used to put the query in apollo's cache
+  await client.query({
+    query: getDatasetsQuery,
+    variables: { where: { workspaceSlug: "local" } },
```

Comment:
Don't forget that we're supposed be able to do this in less than 2 weeks!

@valerian can we do a meeting tomorrow morning where you'd present us what you're doing on #771 (_this would also let us help you getting everything working faster_)?

Reply:
Actually this is a small hack to push something consistent in apollo's cache as my components rely on it

Reply:
OK. Is is possible to use a different workspace name then or is the test still tied to the SW?

Reply:
Yes it's possible, I will use a different name